starboard/shared/win32/decrypting_decoder.cc

```diff
@@ -224,11 +224,13 @@ bool DecryptingDecoder::TryWriteInputBuffer(
 }
 
 bool DecryptingDecoder::ProcessAndRead(ComPtr<IMFSample>* output,
-                                       ComPtr<IMFMediaType>* new_type) {
+                                       ComPtr<IMFMediaType>* new_type,
+                                       bool* hasError) {
```

Comment:
Rename to `has_error` to be in line with the style guide.

Also, is it possible to instead set `error_occurred_` in this function rather than using `hasError`?

Reply:
The error state is returned only for current frame, so I suppose it isn't necessary to hold this state in class variable for further use. I suggest to remain has_error  not  error_occurred_